sdcm/nemesis.py

```diff
@@ -4267,10 +4266,11 @@ def check_ssl_reload_log(node_system_log):
         with context_manager:
             for node in self.cluster.nodes:
                 node_system_logs[node] = node.follow_system_log(
-                    patterns=f'messaging_service - Reloaded {{{ssl_files_location}}}')
-                node.remoter.send_files(src=f'data_dir/ssl_conf/{TLSAssets.DB_CERT}', dst='/tmp')
-                node.remoter.run(f"sudo cp -f /tmp/{TLSAssets.DB_CERT} {ssl_files_location}")
-                new_crt = node.remoter.run(f"cat {ssl_files_location}").stdout
+                    patterns=[f'messaging_service - Reloaded {ssl_cert_location}'])
```

Comment:
put this oneliner change into it's own commit, so we can backport it

Reply:
it's not on it's own PR
https://github.com/scylladb/scylla-cluster-tests/pull/7787